pkg/addons/data/Namespace/openshift-azure.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    openshift.io/node-selector: ""
```

Comment:
Is this needed?

Reply:
This backfired us a few times already (lack of this annotation). I would be tempted to keep it as its default.